static/css/custom.css

```diff
@@ -0,0 +1,465 @@
+
+    --button-primary-color: white;
+}
+
+/* add a transition to ease into new themes */
+/* looks best in Mozilla, text transitions late in Safari/Chrome */
+html.transition-theme,
+html.transition-theme *,
+html.transition-theme *:before,
+html.transition-theme *:after {
+    transition: all 650ms !important;
+    transition-delay: 0 !important;
+}
+
+
+/* ---- General ---- */
+.container {
+    max-width: 800px;
+    margin: 0 auto;
+    padding: 0 10px;
+}
+.docs-header {
+    text-transform: uppercase;
+    letter-spacing: .2rem;
+    font-weight: 600;
+}
+.docs-section {
+    border-top: 1px solid var(--border-color-softer);
+    padding-top: 4rem;
+    padding-bottom: 4rem;
+    margin-bottom: 0;
+}
+ul {
+    margin-left: 2rem;
+}
+.heading-font-size {
+    font-size: 1.2rem;
+    color: #999;
+    letter-spacing: normal;
+}
+@media (min-width: 600px) {
+    .docs-section {
+        padding-top: 6rem;
+        padding-bottom: 6rem;
+    }
+}
+
+
+/* ---- Header ---- */
+.header {
+    margin-top: 1rem;
+    text-align: center;
+}
+.value-prop {
+    margin-top: 1rem;
+}
+.value-props {
+    margin-top: 4rem;
+    margin-bottom: 4rem;
+}
+i.fas, i.fab {
+    font-size: 5rem;
+    display: block;
+    margin: 0 auto 2rem;
+}
+/* grids to 3 columns above mobile sizes */
+@media (min-width: 600px) {
+    .header {
+    margin-top: 2rem;
+    }
+    .value-header {
+        grid-column: 1 / span 3;
+        margin-top: 2rem;
+    }
+    .value-props {
+        margin-top: 9rem;
+        margin-bottom: 7rem;
+    }
+    .value-img {
+        margin-bottom: 1rem;
+    }
+}
+
+/* ---- Nav ---- */
+.navbar {
+    position: sticky;
+    top: 0;
+}
+.navbar .navbar-main {
+    display: flex;
+    justify-content: space-between;
+}
+.navbar + .docs-section {
+    border-top-width: 0;
+}
+.navbar,
+.navbar-spacer {
+    width: 100%;
+    height: 5rem;
+    background: var(--background-color);
+    z-index: 99;
+    border-top: 1px solid var(--border-color-softer);
+    border-bottom: 1px solid var(--border-color-softer);
+}
+.navbar-spacer {
+    display: none;
+}
+/*.navbar > .container {
+    width: 100%;
+}*/
+.navbar-list {
+    list-style: none;
+    margin-bottom: 0;
+}
+.navbar-item {
+    position: relative;
+    margin-bottom: 0;
+}
+.navbar-item img {
+    height: 2rem;
+    vertical-align: middle;
+}
+.navbar-link {
+    text-transform: uppercase;
+    font-size: 1.3rem;
+    font-weight: 600;
+    letter-spacing: .2rem;
+    text-decoration: none;
+    line-height: 5rem;
+    color: var(--text-color-normal);
+}
+.navbar-link.active {
+    color: var(--accent-color);
+}
+
+
+/* ---- Buttons ---- */
+@media (min-width: 600px) {
+    #buttons .grid-container.quarters {
+        grid-template-columns: repeat(2, 1fr);
+    }
+}
+@media (min-width: 900px) {
+    #buttons .grid-container.quarters {
+        grid-template-columns: repeat(4, 1fr);
+    }
+}
+
+/* Images */
+img.feature-icon {
+    height:4rem;
```

Comment:
```suggestion
    height:4rem;
    float: left;
    margin-right: 3rem;
```
looks like this, with `grid-comtainer { align: left; }` above,
though it requires `p, h6 { clear: both; }` and maybe more..
![image](https://user-images.githubusercontent.com/5934199/83554690-7687f180-a50d-11ea-9add-c1e8726789e4.png)